File: Data Structures & Algorithms/find-duplicate-integer/submission-0.cpp
class Solution {
public:
    int findDuplicate(vector<int>& nums) {
        std::unordered_set<int> _set;
        for (int i = 0; i < nums.size(); i++){
            if (_set.count(nums[i]))
                return nums[i];
            _set.insert(nums[i]);
        }
        return -1;
    }
};

File: Data Structures & Algorithms/duplicate-integer/submission-0.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        size = len(nums)
        i = 0
        while i < size - 1:
            j = i + 1
            while j < size:
                if nums[i] == nums[j]:
                    return True
                j += 1
            i += 1
        return False